ompi/mca/coll/libnbc/nbc_ialltoallv.c

```diff
@@ -214,6 +214,7 @@ int ompi_coll_libnbc_ialltoallv_inter (const void* sendbuf, const int *sendcount
   return OMPI_SUCCESS;
 }
 
+__attribute__((unused))
```

Comment:
@hjelmn is right -- we have `__opal_attribute_unused__` as a portable form of `__attribute__((unused))`.  I think this was a bigger deal a while ago, before most (all?) compilers supported the `__attributed__((unused))` form.

...that being said, I guess I actually have no idea which compilers do/do not support `__attributed__((unused))`.  So it's probably safer to use `__opal_attribute_unused__` everywhere (which will be defined to be "" if the compiler doesn't support it).